src/utils/stripUnwantedTags.js

```diff
@@ -0,0 +1,20 @@
+// utils/srtipUnwantedTags
+
+import { parseHTML } from 'linkedom'
+
+/**
+ * @param html {string}
+ * @param exclusions {string[]}
+ * @returns {string}
+ */
+export default (html, exclusions = []) => {
+  if (typeof exclusions !== 'undefined' && !exclusions.length) return html
```

Comment:
```suggestion
  if (!exclusions?.length) return html
```

Take advantage of modern syntax